meta-resin-pyro/README.md

```diff
@@ -0,0 +1,11 @@
+# Resin.io layer for pyro supported boards
```

Comment:
I think that Pyro should be upper cased.

Reply:
I can do this but this readme file is also in the other yocto version directories where lower-case is used so it would need updating there also.

Reply:
Perhaps not necessarily, as they will be removed in a couple of years... :)